store/interfaces.json

```diff
@@ -40,6 +40,7 @@
       "header.full"
     ],
     "allowed": [
+      "product-addon-wrapper",
```

Comment:
I think this could be named differently, currently it's not clear what this does.

Maybe `product-add-to-list-button`?

I think it would have to change the wishlist app as well